2024-06-24 19:42:55,010 INFO    MainThread:36418 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-06-24 19:42:55,010 INFO    MainThread:36418 [wandb_setup.py:_flush():76] Configure stats pid to 36418
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_setup.py:_flush():76] Loading settings from /Users/devu/.config/wandb/settings
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_setup.py:_flush():76] Loading settings from /Users/devu/PycharmProjects/non-parametric-transformers/wandb/settings
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'run.py', 'program_abspath': '/Users/devu/PycharmProjects/non-parametric-transformers/run.py', 'program': 'run.py'}
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_init.py:_log_setup():527] Logging user logs to ./wandb/run-20240624_194255-6jss0yue/logs/debug.log
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_init.py:_log_setup():528] Logging internal logs to ./wandb/run-20240624_194255-6jss0yue/logs/debug-internal.log
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_init.py:init():567] calling init triggers
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {}
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_init.py:init():617] starting backend
2024-06-24 19:42:55,011 INFO    MainThread:36418 [wandb_init.py:init():621] setting up manager
2024-06-24 19:42:55,013 INFO    MainThread:36418 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-06-24 19:42:55,015 INFO    MainThread:36418 [wandb_init.py:init():629] backend started and connected
2024-06-24 19:42:55,026 INFO    MainThread:36418 [wandb_init.py:init():721] updated telemetry
2024-06-24 19:42:55,063 INFO    MainThread:36418 [wandb_init.py:init():754] communicating run to backend with 90.0 second timeout
2024-06-24 19:44:25,385 ERROR   MainThread:36418 [wandb_init.py:init():780] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-06-24 19:44:25,415 ERROR   MainThread:36418 [wandb_init.py:init():1205] Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
Traceback (most recent call last):
  File "/Users/devu/anaconda3/envs/npt/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1187, in init
    run = wi.init()
  File "/Users/devu/anaconda3/envs/npt/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 786, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-06-24 19:44:26,420 WARNING MsgRouterThr:36418 [router.py:message_loop():77] message_loop has been closed
